comparisontool/tests/management/commands/test_load_bah.py

```diff
@@ -0,0 +1,23 @@
+import tempfile
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from comparisontool.models import BAHRate
+
+
+class LoadBAHTests(TestCase):
+    def test_load_csv(self):
```

Comment:
Nitpicky, but would it make sense to call this `test_load_bah` to remain consistent with the command name?  Or if you want to be more descriptive, `test_BAHRate_objects_created_successfully_from_csv`  ?